lib/security/config.pm

```diff
@@ -0,0 +1,20 @@
+# openssl fips test
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Summary: Stronger password to be used with CC/FIPS.
+#
+# Maintainer: QE Security <none@suse.de>
+
+package security::config;
+
+use strict;
+use warnings;
+
+use base 'Exporter';
+
+our @EXPORT = qw(strong_password);
+
+our $strong_password = "not-so-s3cr3t_BUT_VERY_STRONG.";
```

Comment:
If this is going to be hardcoded, please do make sure that there is some POD documentation about it that can be shown at https://os-autoinst.github.io/os-autoinst-distri-opensuse/registration.html

```suggestion
our $strong_password = "not-so-s3cr3t_BUT_VERY_STRONG.";
```

Reply:
ok, will do that!